yoga/Yoga.c

```diff
@@ -1138,38 +1289,38 @@ static void YGNodeAbsoluteLayoutChild(const YGNodeRef node,
       childHeight = node->layout.measuredDimensions[YGDimensionHeight] -
                     (YGNodeLeadingBorder(node, YGFlexDirectionColumn) +
                      YGNodeTrailingBorder(node, YGFlexDirectionColumn)) -
-                    (YGNodeLeadingPosition(child, YGFlexDirectionColumn) +
-                     YGNodeTrailingPosition(child, YGFlexDirectionColumn));
-      childHeight = YGNodeBoundAxis(child, YGFlexDirectionColumn, childHeight);
+                    (YGNodeLeadingPosition(child, YGFlexDirectionColumn, width) +
```

Comment:
Position percentages are calculated based on the same axis as the position (https://jsfiddle.net/jckmwztt/15/) this is different from margin and padding and instead similar to width and height. So this (and other locations) should be `YGFlexDirectionColumn, height`